newIDE/app/src/ProjectsStorage/LocalProjectWriter.js

```diff
@@ -80,6 +134,38 @@ export default class LocalProjectWriter {
     }
   };
 
+  static saveProjectAs = (project: gdProject): Promise<void> => {
+    const filepath = project.getProjectFile();
+    const filenameWithExtension = path.basename(project.getProjectFile()) || "game.json";
+    const projectPath = path.dirname(project.getProjectFile());
+    const fileSystem = assignIn(new gd.AbstractFileSystemJS(), localFileSystem);
+    const browserWindow = electron.remote.getCurrentWindow();
+    const options = {
+    defaultPath: path.join(projectPath, filenameWithExtension),
+    filters: [{ name: 'GDevelop 5 project', extensions: ['json'] }],
+    };
+
+    if (!dialog) {
+    return Promise.reject('Unsupported');
+    }
+    const newFilepath = dialog.showSaveDialog(browserWindow, options);
+    if (!newFilepath || newFilepath === '') {
+    return Promise.reject('Filepath from dialog is empty');
+    }
+    const newProjectFolder = path.dirname(newFilepath);
+
+    gd.ProjectResourcesCopier.copyAllResourcesTo(
```

Comment:
nitpicking: don't forget to run prettier (npm run format) (or using Prettier extension for VSCode)